src/binaryen-c.cpp

```diff
@@ -190,41 +190,40 @@ void BinaryenModuleDispose(BinaryenModuleRef module) {
 
 // Function types
 
-BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams) {
+BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name,
```

Comment:
please don't change the indentation (we don't have a strict length limit here)

Reply:
Yes, you do: https://github.com/WebAssembly/binaryen/blob/master/.clang-format#L4